.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,84 @@
+name: build and publish container image
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - master      
```

Comment:
Hmm, so the `latest` tag will would flip between stable release versions and master builds. I think this is confusing.